src/components/app/index.jsx

```diff
@@ -0,0 +1,2 @@
+import AppContainer from './app-container.jsx';
+export default AppContainer;
```

Comment:
```suggestion

export default AppContainer;
```
И пустую строку добавить в конце!